hera_sim/simulate.py

```diff
@@ -267,10 +293,21 @@ def write_data(self, filename, file_type="uvh5", **kwargs):
                 :class:`pyuvdata.UVData`) which determines which write method to call.
             **kwargs: keyword arguments sent directly to the write method chosen.
         """
+        ret_seeds = kwargs.pop('ret_seeds', False)
```

Comment:
can we just make these arguments, rather than `kwargs`? I don't see a good reason to leave them as kwargs.

Reply:
I would think that it would be better to put them in as named kwargs, rather than something that might be stored in `kwargs` (i.e. including `ret_seeds=False` in the function signature). Is that what you meant?